code/modules/mob/living/simple_animal/abnormality/waw/luna.dm

```diff
@@ -0,0 +1,184 @@
+	icon = 'ModularTegustation/Teguicons/96x48.dmi'
+	icon_state = "dellaluna"
+	maxHealth = 400
+	health = 400
+	start_qliphoth = 3
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(20, 30, 40, 50, 55),
+		ABNORMALITY_WORK_INSIGHT = list(40, 45, 50, 55, 55),
+		ABNORMALITY_WORK_ATTACHMENT = list(30, 30, 50, 50, 55),
+		ABNORMALITY_WORK_REPRESSION = 30,
+		"Performance" = 70
+		)
+	pixel_x = -32
+	base_pixel_x = -32
+	work_damage_amount = 10
+	work_damage_type = WHITE_DAMAGE
+	max_boxes = 20
+	ego_list = list(
+		/datum/ego_datum/weapon/moonlight,
+		/datum/ego_datum/armor/moonlight
+		)
+//	gift_type =  /datum/ego_gifts/moonlight
+	var/performance = FALSE
+	var/performance_length = 60 SECONDS
+	var/breach_length = 404 SECONDS		//How long the song is (when I finally finish it)
+	var/breached = FALSE
+	var/breached_monster
+	var/killspawn
+
+/mob/living/simple_animal/hostile/abnormality/luna/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(50))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/luna/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/luna/ZeroQliphoth(mob/living/carbon/human/user)
+	icon_state = "dellaluna_breach"
+	var/turf/W = pick(GLOB.department_centers)
+	var/mob/living/simple_animal/hostile/luna/spawningmonster = new(get_turf(W))
+	breached_monster = spawningmonster
+	breached = TRUE
+	addtimer(CALLBACK(src, .proc/BreachEnd, user), breach_length)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/luna/WorkComplete(mob/living/carbon/human/user, work_type, pe)
+	if(work_type == "Performance")
+		datum_reference.qliphoth_change(-1)
+		if(get_attribute_level(user, PRUDENCE_ATTRIBUTE) < 60)
+			user.adjustSanityLoss(-500) // It's not stated in game but performing with level 3 prudence and lower make them instantly panic
+
+		//and half your HP.
+		user.adjustBruteLoss(user.maxHealth*0.5)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/luna/Worktick(mob/living/carbon/human/user, work_type)
+	if(performance)
+		user.apply_damage(work_damage_amount, BLACK_DAMAGE, null, user.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+
+
+/mob/living/simple_animal/hostile/abnormality/luna/AttemptWork(mob/living/carbon/human/user, work_type)
+	if(work_type == "Performance")
+		to_chat(user, "<span class='nicegreen'>Please wait until the performance is completed.</span>")
+		addtimer(CALLBACK(src, .proc/PerformanceEnd, user), performance_length)
+		for(var/mob/living/carbon/human/L in GLOB.player_list)
+			L.apply_status_effect(STATUS_EFFECT_LUNAR)
+
+		if(!performance)
+			performance = TRUE
+			playsound(user, 'sound/abnormalities/luna/moonlight_sonata.ogg', 100, FALSE, 28)	//Let the people know.
+
+		if(breached)	//You will have to start a new performance to delete the breached abno.
+			killspawn = TRUE
+	return TRUE
```

Comment:
Specifically could they walk off to short-stop the downside of the Performance? If so, should there be a larger punishment for doing so?

Reply:
Nah, still fires due to work complete.